youtube_dl/extractor/streetvoice.py

```diff
@@ -3,47 +3,50 @@
 
 from .common import InfoExtractor
 from ..compat import compat_str
-from ..utils import unified_strdate
 
 
 class StreetVoiceIE(InfoExtractor):
     _VALID_URL = r'https?://(?:.+?\.)?streetvoice\.com/[^/]+/songs/(?P<id>[0-9]+)'
     _TESTS = [{
-        'url': 'http://streetvoice.com/skippylu/songs/94440/',
-        'md5': '15974627fc01a29e492c98593c2fd472',
+        'url': 'https://streetvoice.com/lekimberleyy/songs/630281/',
+        'md5': '515d4bfdfe337039d9efaa53be24484d',
         'info_dict': {
-            'id': '94440',
+            'id': '630281',
             'ext': 'mp3',
-            'title': '輸',
-            'description': 'Crispy脆樂團 - 輸',
-            'thumbnail': r're:^https?://.*\.jpg$',
-            'duration': 260,
-            'upload_date': '20091018',
-            'uploader': 'Crispy脆樂團',
-            'uploader_id': '627810',
+            'title': '郵票Good Times feat. 瘦子E.SO',
+            'description': 'Kimberley - 郵票Good Times feat. 瘦子E.SO',
+            'thumbnail': r're:^https?://.*\.jpg',
+            'uploader': 'lekimberleyy',
+            'uploader_id': '2674557',
         }
     }, {
-        'url': 'http://tw.streetvoice.com/skippylu/songs/94440/',
+        'url': 'https://tw.streetvoice.com/ConstantanChange/songs/628471/',
         'only_matching': True,
     }]
 
     def _real_extract(self, url):
         song_id = self._match_id(url)
 
+        song_details = self._download_json(
+            'https://streetvoice.com/api/v4/song/%s/?fields=id,name,image,likes_count,is_like,user,profile,nickname,type,public,username' % song_id, song_id, note="Downloading song details")
+
         song = self._download_json(
-            'https://streetvoice.com/api/v1/public/song/%s/' % song_id, song_id, data=b'')
+            'https://streetvoice.com/api/v4/song/%s/hls/file/' % song_id, song_id, data=b'', note="Downloading song data")
```

Comment:
extract all formats.

Reply:
I don't see any other formats and the json response isn't even an array I can iterate, seems like they only have one format even in the old api